modules/products/bamboo/variables.tf

```diff
@@ -49,7 +49,10 @@ variable "installation_timeout" {
 variable "termination_grace_period" {
   description = "Termination grace period in seconds"
   type        = number
-  default     = 30
+  validation {
+    condition     = var.termination_grace_period > 0
```

Comment:
Can't the grace period be `0`? 
I suggest to set a range to something like [0, 5 minutes]

Reply:
The very purpose of this change is to be able to make it 0 :) Let me fix it